frontend/src/pages/TaskOverviewPage.tsx

```diff
@@ -190,9 +228,8 @@ const TaskOverview: FC<{
         name={task.name}
         previousAndNext={siblingTasks}
         onOverviewChange={(id) => history.push(`${tasksPage}/${id}`)}
-        onDataEditConfirm={handleEditConfirm}
         key={task.id}
-        {...getTaskOverviewData(task, hasEditPermission, users!)}
+        {...getTaskOverviewData(task, hasEditPermission, users!, roadmapId)}
```

Comment:
The `users` could also be checked at the call site, or alternatively the function could take optional users (which might be nicer as the users aren't so important for it).